beacon-chain/sync/validate_committee_index_beacon_attestation.go

```diff
@@ -0,0 +1,94 @@
+package sync
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/p2p"
+	"github.com/prysmaticlabs/prysm/shared/bls"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/traceutil"
+	"go.opencensus.io/trace"
+)
+
+// Validation
```

Comment:
Concerned on lack of testing for `validateCommitteeIndexBeaconAttestation`. Cases such as `data.Slot` within range, `attestation` is unaggregated, and `index` is for correct subnet are good unit tests candidates.
If this PR needs to be merged asap, please open an issue for `validateCommitteeIndexBeaconAttestation ` testing, I'll approve it then